db/virtual_tables.cc

```diff
@@ -175,7 +175,7 @@ class token_ring_table : public streaming_virtual_table {
             _db.get_keyspaces()
                 | boost::adaptors::filtered([] (auto&& e) {
                       auto&& rs = e.second.get_replication_strategy();
-                      return rs.is_vnode_based();
+                      return rs.is_vnode_based() || rs.uses_tablets();
```

Comment:
This is "return true", no need to filter.

Reply:
It still filters out local replication strategies.

Reply:
@denesb is there a function that checks exactly what you wanted (local replication strategy)? I agree with Avi is very confusing to see is_vnode_based || uses_tablets, which at first glance seems is always "true" (it seems that every table should be based either on vnodes or tablets).

Reply:
I used `database::get_non_local_strategy_keyspaces()` in v2.